Client/Frontend/Browser/OpenSearch.swift

```diff
@@ -236,11 +236,17 @@ class OpenSearchParser {
                         }
 
                         let name = paramIndexer.attributes["name"]
-                        let value = paramIndexer.attributes["value"]
+                        var value = paramIndexer.attributes["value"]
                         if name == nil || value == nil {
                             print("Param element must have name and value attributes", terminator: "\n")
                             return nil
                         }
+
+                        // Change partner code (pc) for Bing search on iPad (quick fix)
+                        if name == "pc", shortName == "Bing", UIDevice.current.userInterfaceIdiom == .pad {
+                            value = "MOZL"
+                        }
+
```

Comment:
I agree it's not the best solution @thatswinnie but if are to add this then let's also put the ticket in the reference comment as well (see above)